syntax = "proto3";

message Env_data_frame {
    int32 ID          = 1;
    int32 time_stamp  = 2;
    int32 seq_num     = 3;
    float temperature = 4;
    float co2_level   = 5;
    float humidity    = 6;
    int32 crc         = 7;
}

message Env_cmd_frame {
    int32 ID     = 1;

    enum cmd {
        START = 0;
        PING  = 1;
    }

    int32 crc    = 2;
}

message Env_ack_frame {
    int32  ID    = 1;

    enum ack {
        ACK = 0;
        NCK = 1;
    }

    int32 crc    = 2;
}

